google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2015 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""User friendly container for Cloud Bigtable Backup."""
+
+import re
+# import warnings
+# from datetime import datetime
+
+from google.cloud._helpers import _datetime_to_pb_timestamp
+from google.cloud.bigtable_admin_v2.gapic import enums
+# from google.cloud.bigtable_admin_v2.proto import table_pb2
+# from google.cloud.bigtable_admin_v2.proto import bigtable_table_admin_pb2
+from google.cloud.exceptions import NotFound
+
+_BACKUP_NAME_RE = re.compile(
+    r"^projects/(?P<project>[^/]+)/"
+    r"instances/(?P<instance_id>[a-z][-a-z0-9]*)/"
+    r"clusters/(?P<cluster_id>[a-z][-a-z0-9]*)/"
+    r"backups/(?P<backup_id>[a-z][a-z0-9_\-]*[a-z0-9])$"
+)
+_DEFAULT_EXPIRE_PERIOD = 604800 # 7 days
+
+
+class Backup(object):
+	""" Representation of a Google Cloud Bigtable Backup.
+
+	A :class`Backup` can be used to:
+
+	* :meth:`create` the backup
+	* :meth:`update` the backup
+	* :meth:`delete` the backup
+
+	:type backup_id: str
+	:param backup_id: The ID of the backup.
+
+	:type instance: :class:`~google.cloud.spanner_v1.instance.Instance`
+	:param instance: The Instance that owns the Backup.
+
+	:type cluster_id: str
+	:param cluster_id: (Optional) The ID of the Cluster that contains this Backup.
+					   Required for calling 'delete', 'exists' etc. methods.
```

Comment:
Indent: "Required" should start right under "(Optional)"

Reply:
Yes, I keep getting this kind of stuff, despite that it looks right in my PyCharm:
![image](https://user-images.githubusercontent.com/38331387/84906933-94626400-b080-11ea-9c11-5e5eb37d9cc8.png)